evm_loader/program/src/executor.rs

```diff
@@ -347,7 +347,36 @@ impl<'config, B: Backend> Machine<'config, B> {
         let context = evm::Context{address: code_address, caller: caller, apparent_value: U256::zero()};
 
         let runtime = evm::Runtime::new(Rc::new(code), Rc::new(input), context, &self.executor.config);
-        self.runtime.push((runtime, CreateReason::Root));
+        self.runtime.push((runtime, CreateReason::Call));
+    }
+
+    pub fn create_begin(&mut self, caller: H160, code: Vec<u8>, gas_limit: u64){
+
+        let scheme = evm::CreateScheme::Legacy {
+            caller: caller,
+        };
+        self.executor.state.enter(u64::max_value(), false);
+
+        match self.executor.create(caller, scheme, U256::zero(),&code, None ){
+            Capture::Exit((reason, address, return_data)) => {
+                debug_print!("create_begin() error ");
+            },
```

Comment:
Return an error. And check it in the `entrypoint.rs`.

Reply:
comments was applied